packages/elements/src/datetime-picker/index.ts

```diff
@@ -1088,6 +1093,46 @@ export class DatetimePicker extends FormFieldElement implements MultiValue {
     this.validateInput();
   }
 
+  /**
+   * Run when tap event happens on clears button and fire value-changed event.
+   * @param event tap event
+   * @returns {void}
+   */
+  protected override onClearsButtonTap(event: TapEvent): void {
+    let values: string[] = [];
+    if (this.inputFromEl && this.inputToEl) {
+      // range mode
+      this.inputFromEl.value = '';
+      this.inputToEl.value = '';
+      // When the element's input is cleared by typing,
+      // an array of empty string(s) will be assign to `values`.
+      // Clear button logic simply follows this behavior for consistency.
+      // TODO: `values` should be set to an empty array once the input has been cleared.
+      values = ['', ''];
+    } else if (this.inputEl) {
+      // single mode
+      this.inputEl.value = '';
+      values = [''];
+    }
+    this.notifyValuesChange(values);
+    this.inputValues = values;
+    this.resetViews();
+    // validateInput and request render for the cases where values do not change.
+    this.validateInput();
+    this.requestUpdate();
+    event.preventDefault();
```

Comment:
Let's add a comment here as it doesn't call super class's method.
```suggestion
    // // PreventDefault() ensures that selecting the clear button would not change `opened` state.
    event.preventDefault();
```